plugins/org.eclipse.n4js.transpiler.es/src/org/eclipse/n4js/transpiler/es/transform/ExpressionTransformation.xtend

```diff
@@ -35,15 +31,22 @@ import org.eclipse.n4js.ts.types.TInterface
 import org.eclipse.n4js.ts.types.TMethod
 import org.eclipse.n4js.ts.utils.TypeUtils
 import org.eclipse.n4js.utils.PromisifyHelper
-import java.util.ArrayList
 
 import static org.eclipse.n4js.transpiler.TranspilerBuilderBlocks.*
 
 import static extension org.eclipse.n4js.typesystem.RuleEnvironmentExtensions.*
 
 /**
  * Some expressions need special handling, this is done in this transformation.
+ * <p>
+ * Dependencies:
+ * <ul>
+ * <li>ExcludesBefore: AsyncAwaitTransformation<br>
```

Comment:
as discussed:
1. naming of these annotations was not changed in this PR and
2. naming these dependencies is a bit tricky and there are always pro/cons for all options.

If we want to change that, we need to do it as part of a dedicated enhancement issue. But in that case we have to carefully check all annotations and cases.